CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # AWS AppSync SDK for iOS - CHANGELOG
 
-The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and perform operations like `Queries`, `Mutations` and `Subscriptions`. The SDK also includes support for offline operations.
+The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and perform operations like `Queries`, `Mutations` and `Subscriptions`. The SDK
+also includes support for offline operations.
+
+## 2.11.1
+
+### Bug Fixes
+- Updated SQLite.swift to master to fix caching error. ([Issue #211](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/211))
+
+### Misc. Updates
+- Updated AWS SDK dependencies to 2.9.5
+- Updated SQLite.swift to the latest version of the master branch instead of official release. The latest official release of SQLite.swift does not work
+  with apps compiled in Xcode 10.2 because it relies on the format of the `#function` expression behavior, which changed between 10.1 and 10.2. This
```

Comment:
Are the line breaks caused due to some IDE setting? Not sure of the rendering due to new lines.

Reply:
That's intentional, I'm inserting hard line breaks into the long lines of text to make reviews & editing easier in text editors like vi.